### PR TITLE
docs: corrected typos, grammatical errors & fixed broken URL

### PR DESCRIPTION
**Pull request by Neilblaze**
#### Summary
A few typos and grammatical errors were discovered and have been corrected. Additionally, a broken URL has been fixed.

The broken [URL](https://content.nuxtjs.org/themes-docs) can be explored [here](https://github.com/sigstore/docs#:~:text=%40nuxt/content%20theme%20docs) & it returns **404**, but fortunately, the same for the **_[Japanese variant](https://content.nuxtjs.org/ja/v1/examples/docs-theme/)_** works. I've raised an [issue](https://github.com/nuxt/content/issues/1987) in `nuxt/content` for the same.

#### Checklist

- [x] I have explained the need for this PR and the problem it solves.
- [x] I have explained the changes or the new features added to this PR.
- [x] I have updated the documentation to reflect this change.
- [x] I have verified that this change is backward compatible.

cc: @ltagliaferri

### CODE REVIEW

**Review by cpanato — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,4 +48,4 @@ yarn generate
 
 To preview the static generated app, run `yarn start`
 
-For detailed explanation on how things work, checkout [nuxt/content](https://content.nuxtjs.org) and [@nuxt/content theme docs](https://content.nuxtjs.org/themes-docs).
+For detailed explanation on how things work, checkout [nuxt/content](https://github.com/nuxt/content).
```
Comment (ltagliaferri) [inline README.md:51]: I would keep both the frontend site https://content.nuxtjs.org and add the GitHub here

File: content/en/api-stability.md [hunk 2]
```diff
@@ -34,10 +34,10 @@ There are three levels of stability and support:
 
 **Beta**: Backwards incompatible changes require at least 2 months notice
 
-**Generally Available**: Deprecating features requries at least 6 months notice
+**Generally Available**: Deprecating features require at least 6 months notice
```
Comment (ltagliaferri) [inline content/en/api-stability.md:37]:
```suggestion
**Generally Available**: Deprecating features requires at least 6 months notice
```

File: content/en/contributing.md [hunk 3]
```diff
@@ -23,13 +23,13 @@ There’s plenty of good style guides for writing technical documentation. [Micr
 
 - Using insets, alert banners and callouts help make each page a little more visually interesting, which helps a reader engage with, follow and understand what you’re writing about.
 
-- Always check your assumptions. It’s easy to assume your reader knows exactly what you do. But that’s not always the case. Be careful writing about anything that might require more context, embed links in passing if it’ll help. If you’re introducing something difficult for the first time, state it, explain it as simply as you can, and then move to its application using consistent language - don’t be tempted to start using variations or introducing more tangents than is absolutely critical.
+- Always check your assumptions. It’s easy to assume your reader knows exactly what you do. But that’s not always the case. Be careful writing about anything that might require more context, and embed links in passing if it’ll help. If you’re introducing something difficult for the first time, state it, explain it as simply as you can, and then move to its application using consistent language - don’t be tempted to start using variations or introducing more tangents than is absolutely critical.
```
Comment (ltagliaferri) [inline content/en/contributing.md:26]: The `and` is not necessary here

File: content/en/cosign/git_support.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ it is `github://`.
 
 ### Key Generation and Management
 
-To generate keys using a Git provider, you can use the `cosign generate-key-pair` command with the as the last
+To generate keys using a Git provider, you can use the `cosign generate-key-pair` command with the last
```
Comment (ltagliaferri) [inline content/en/cosign/git_support.md:21]:
```suggestion
To generate keys using a Git provider, you can use the `cosign generate-key-pair` command as the last
```

File: content/en/cosign/hardware-based-tokens.md [hunk 5]
```diff
@@ -178,10 +178,10 @@ $ go test ./test -tags=resetyubikey,e2e -count=1
 There are two attestations available from the hardware key.
 The first is the device attestation.
 This can be used to verify the hardware is authentic and came from the manufacturer.
-To verify this, retrieve the manufacturers CA.
+To verify this, retrieve the manufacturer's CA.
 See [here](https://developers.yubico.com/yubico-piv-tool/Attestation.html) for instructions from Yubico.
 
-This certificate can be validated with `openssl` or other tooling:
+This certificate can be validated with `openssl` or another tooling:
```
Comment (ltagliaferri) [inline content/en/cosign/hardware-based-tokens.md:184]: Keep, so the meaning is not changed

```suggestion
This certificate can be validated with `openssl` or other tooling:
```

File: content/en/index.md [hunk 6]
```diff
@@ -30,7 +30,7 @@ Your certificate then comes back to Sigstore, where Sigstore exchanges keys, ass
 
 ## Software supply chain security
 
-Software supply chains are exposed to multiple risks. Users are susceptible to various targeted attacks, along with account and cryptographic key compromise. Keys in particular are a challenge for software maintainers to manage. Projects often have to maintain a list of current keys in use, and manage the keys of individuals who no longer contribute to a project. Projects all too often store public keys and digests on git repo readme files or websites, two forms of storage susceptible to tampering and less than ideal means of securely communicating trust.
+Software supply chains are exposed to multiple risks. Users are susceptible to various targeted attacks, along with account and cryptographic key compromises. Keys in particular are a challenge for software maintainers to manage. Projects often have to maintain a list of current keys in use, and manage the keys of individuals who no longer contribute to a project. Projects all too often store public keys and digests on git repo readme files or websites, two forms of storage susceptible to tampering and less than ideal means of securely communicating trust.
```
Comment (ltagliaferri) [inline content/en/index.md:33]: I would keep this as it somewhat changes the meaning
```suggestion
Software supply chains are exposed to multiple risks. Users are susceptible to various targeted attacks, along with account and cryptographic key compromise. Keys in particular are a challenge for software maintainers to manage. Projects often have to maintain a list of current keys in use, and manage the keys of individuals who no longer contribute to a project. Projects all too often store public keys and digests on git repo readme files or websites, two forms of storage susceptible to tampering and less than ideal means of securely communicating trust.
```

File: content/en/policy-controller/overview.md [hunk 7]
```diff
@@ -53,7 +53,7 @@ kubectl label namespace my-excluded-namespace policy.sigstore.dev/exclude=true
 
 *Note* If you change the behaviour to opt-out, it might cause some grief in
 system namespaces, so be mindful. Also, consider looking at changing the default
-behaviour of when an image is not matched below and while rolling things out,
+behaviour when an image is not matched below and while rolling things out,
```
Comment (ltagliaferri) [inline content/en/policy-controller/overview.md:56]: This should be stetted as it changes the meaning 

```suggestion
behaviour of when an image is not matched below and while rolling things out,
```

File: content/en/rekor/verify-release.md [hunk 8]
```diff
@@ -90,13 +90,13 @@ email:ctadeu@gmail.com
 
 OK, so we can see that a Fulcio-generated certificate, with a chain to the root certificate has the email identity of a sigstore maintainer (in this instance, our own release manager the awesome Carlos).
 
-If you wanted to besuper paranoid, you could email Carlos and ask him to give you assurance he is Carlos and he signed the release.
+If you wanted to be super paranoid, you could email Carlos and ask him to give you assurance he is Carlos and he signed the release.
```
Comment (ltagliaferri) [inline content/en/rekor/verify-release.md:93]: Let's remove this joke

File: content/en/rekor/verify-release.md [hunk 9]
```diff
@@ -90,13 +90,13 @@ email:ctadeu@gmail.com
...
 
 #### Verify the entry is in the transparency log
 
 On the  [release page](https://github.com/sigstore/rekor/releases) you will see some URLs, these are the rekor entries of the signing events.
 
-This is the link for the binary and signing materials we have been working with in this guide
+This is the link for the binary and signing materials we have been working within this guide
```
Comment (ltagliaferri) [inline content/en/rekor/verify-release.md:99]: Stet or rewrite

```suggestion
This is the link for the binary and signing materials we have been working with throughout this guide
```

File: content/en/rekor/verify-release.md [hunk 10]
```diff
@@ -140,7 +140,7 @@ $ curl -X GET "https://rekor.sigstore.dev/api/v1/log/entries/b6fdc91e6af5bdd8df1
 }
 ```
 
-Woah, that's a lot of stuff. Let's focus on the the body and decode it from base64:
+Woah, that's a lot of stuff. Let's focus on the body and decode it from base64:
```
Comment (ltagliaferri) [inline content/en/rekor/verify-release.md:143]: Rewrite to remove the joke

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thanks for making some updates here. Some of the changes do modify the meaning of the text so some should be revered. As for the jokes that already exist in the documentation, those should be removed rather than grammatically corrected as we should aim to make the documentation more objective so that it is easier for readers to understand

File: README.md (hunk 1 above)
Comment (Neilblaze) [reply README.md:51]: So, something like this? 👇🏼 

For detailed explanation on how things work, checkout [nuxt/content](https://content.nuxtjs.org) & it's [github repository](https://github.com/nuxt/content).

Comment (ltagliaferri) [reply README.md:51]:
```suggestion
For detailed explanation on how things work, checkout [nuxt/content](https://github.com/nuxt/content) and their official [GitHub repo](https://github.com/nuxt/content).
```

**Comment by Neilblaze** [comment]
Technically markdown doesn't consider "new line" directly without breaks (<br/>), I still fixed those two broken passages as it was looking a bit odd. 😅

**Review by ltagliaferri — APPROVED** [review]
Thank you for the help, lots of important fixes! I'll take a look at the open `TODO`s later this week

**Pull request merged**